IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_Stationing/DocPropertySet.xml

```diff
@@ -3,6 +3,8 @@
 	<Properties>
 		<DocProperty xsi:nil="true" href="IncomingStation_2ScYZDmQ9C9vR878Q0zliC" />
 		<DocProperty xsi:nil="true" href="Station_2wUFXsqVDB8fwdTk7zGJXN" />
+		<DocProperty xsi:nil="true" href="OffsetLateralDistance_3YfGFWfmbEBARcr6VLI5x" />
```

Comment:
How is this still a station? Now we are talking about a position already. Is there any documentation we could reference?

We should not confuse concepts. If all of this is supposed to be in the pset, then we are talking about a Pset_PositionExpression.